deps/wazuh_testing/wazuh_testing/tools/wazuh_manager.py

```diff
@@ -0,0 +1,30 @@
+import os
+import subprocess
+import subprocess as sb
```

Comment:
```suggestion
import subprocess as sb
```

It is imported twice, we can delete one import if it is not necessary.

Reply:
Done d2419f50de93aa3c8c7ce0968c6abbe81321f316